run.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import getpass
+import time
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+from palantiri.core import engine, crawler, datahandler
```

Comment:
This is the 3rd example program using palantiri. We should probably remove some or move them to `/examples` or something like that.

Reply:
:+1: bad case of copy-pasta